core-plugins/docs/OracleExport-action.md

```diff
@@ -0,0 +1,70 @@
+# Oracle Export Action
+
+
+Description
+-----------
+A plugin that can efficiently export data from Oracle to be used in Hydrator pipelines.
+Oracle includes command line tools to export data that can be utilized to perform this task.
+
+
+Use Case
+--------
+A Hydrator user would like to export oracle data onto hdfs or local file system using an action plugin
+that does not require a JDBC connection to perform the export from Oracle.
+
+
+Properties
+----------
+
+**oracleServerHostname:** Hostname of the remote DB machine.
+
+**oracleServerPort:** Port of the remote DB machine.Defaults to 22.
```

Comment:
Also, better to mention that this is the SSH port for the Oracle host (host instead of machine).